framework/src/modules/http_api/controllers/accounts.js

```diff
@@ -254,8 +249,7 @@ AccountsController.getMultisignatureMemberships = async function(
 
 	try {
 		let groups = await storage.entities.Account.get(
-			{ membersPublicKeys_in: [account.publicKey] },
-			{ extended: true },
+			{ membersPublicKeys_in: [`"${account.publicKey}"`] }, // Need to add quotes for PSQL array search
```

Comment:
It seems we never use this filter as an array. If that is true, would it be easier to only have the filter `membersPublicKeys` and avoid the need of quotes here?

Reply:
Just tested this and even when not using array you need to provide the quotes.

Reply:
Changed to `membersPublicKeys` accepting a single PK. Quotes need to be added anyway but the query it's simpler now.

Reply:
Try using `'mem_accounts."membersPublicKeys" @> ${membersPublicKeys:name}'` and remove the quotes from the call

Reply:
@lsilvs trying that throws the error `pgp_adapter.js:71 in error): column "141b16ac8d5bd150f16b1caa08f689057ca4c4434445e56661831f4e671b7c0" does not exist (module=http_api:database, err.code=42703)
    error: column "141b16ac8d5bd150f16b1caa08f689057ca4c4434445e56661831f4e671b7c0" does not exist`